### PR TITLE
Kata skeleton for python

### PR DESCRIPTION
**Pull request by alexandrajulius**
Feel free to test and run it, hope it works :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,27 @@ Run the specs:
...
+```
+$ cd python
+```
+
+Install, create and activate the virtualenv:
+
+```
+$ pip3 install virtualenv
+$ virtualenv -p python3 venv
+$ . venv/bin/activate
```
Comment (dantleech) [inline README.md:89]:
```
./venv/bin/activate`
chmod a+x ./ven/bin/activate
```

File: README.md [hunk 2]
```diff
@@ -75,6 +75,27 @@ Run the specs:
...
+$ pip install -r requirements.txt
```
Comment (dantleech) [inline README.md:90]: should probably be `pip3` as with the first `pip` ?

File: README.md [hunk 3]
```diff
@@ -75,6 +75,27 @@ Run the specs:
...
+```
+
+Run the tests:
+
+```
+$ venv/bin/pytest -vvv
```
Comment (dantleech) [inline README.md:96]: `./venv/bin/pytest` (or without the `chmod` we would need to prefix it with `python3` probably)

Comment (dantleech) [reply README.md:96]: but my `pytest` isn't available :thinking:

Comment (dantleech) [reply README.md:96]: it works globally however, `$ pytest` not sure how that works

Comment (dantleech) [reply README.md:96]: also `python3 -m pytest` works

File: README.md (hunk 2 above)
Comment (alexandrajulius) [reply README.md:90]: it installs the libraries inside the virtual env, no need for `pip3` here

Comment (dantleech) [reply README.md:90]: the `activate` should be `source ./venv/bin/activate` it seems

**Pull request merged**